cumulusci/tasks/bulkdata.py

```diff
@@ -781,6 +899,7 @@ def _fields_for_mapping(self, mapping):
             )
         return fields
 
+>>>>>>> master
```

Comment:
@jlantz bad merge

Reply:
Oops, apparently I didn't save the conflict resolution before commit